pkgs/development/libraries/nv-codec-headers/default.nix

```diff
@@ -1,22 +1,28 @@
-{ lib, stdenv, fetchgit }:
+{ lib
+, fetchgit
+, stdenvNoCC
+}:
 
-stdenv.mkDerivation rec {
-  pname = "nv-codec-headers";
-  version = "9.1.23.1";
-
-  src = fetchgit {
-    url = "https://git.videolan.org/git/ffmpeg/nv-codec-headers.git";
-    rev = "n${version}";
-    sha256 = "1xfvb3mhz6wfx9c732888xa82ivaig903lhvvrqqzs31qfznsplh";
+let
+  make-nv-codec-headers = (import ./make-nv-codec-headers.nix) {
```

Comment:
Shouldn't this file be called generic.nix or common.nix?

Reply:
_Data maxima venia_, such name for a file is ridiculous and convey no serious, palpable meaning.
Something like `make-XYZ` makes clear that this file contains a function (or something like that) that makes `XYZ`.